src/scenekit.cs

```diff
@@ -2498,35 +2502,28 @@ interface SCNProgram : NSCopying, NSSecureCoding {
 	[BaseType (typeof (NSObject))]
 	[Model, Protocol]
 	interface SCNProgramDelegate {
-#if MONOMAC
-	#if XAMCORE_3_0
```

Comment:
I don't believe removing XAMCORE_3_0 is a safe transformation.

Reply:
we already have XAMCORE_3_0, right? Since NET == XAMCORE_4_0, The if was just around a NoiOS, so it gives no harm since the method was not there.

Reply:
I removed the `#if XAMCORE_3_0`'s because I believe the compiler would not let me add another [NoiOS] outside of it (since it had duplicate attributes) and when I didn't add it, xtro would complain that things were being set for iOS but were not really there. I could be misremembering the above though. 
I figured that if I have to add a `#if !XAMCORE_3_0` to add the [NoiOS] there as well and the fact that it was already inside a `#if MONOMAC` and not being able to be called for iOS anyways, that it would maybe be okay to remove.

Any thoughts on this? @mandel-macaque @chamons

Reply:
That's some convoluted code... in general removing XAMCORE_3_0 isn't a safe transformation, but in this particular case, the code inside the XAMCORE_3_0 condition didn't do anything, because it was also inside a `MONOMAC` condition, thus the XAMCORE_3_0 condition is safe to remove.